src/mame/heathkit/intr_cntrl.cpp

```diff
@@ -31,6 +31,11 @@ void heath_intr_cntrl::device_start()
 	m_intr_lines = 0;
 }
 
+void heath_intr_cntrl::device_reset()
+{
+	m_intr_lines = 0;
+}
+
```

Comment:
This value comes from inputs.  You shouldn’t change it on reset.  If the devices that produce those inputs deassert their outputs on reset, it’s their responsibility to update the outputs on reset.